document store not franchised (#50)
wesbos/dad-jokes: readme.md

**A:** Because he used up all his cache

---

> **Q:** Why did the document store database not get franchised?

Reviewer: ```suggestion
**Q:** Why did the document store go out of business?
```

> **A:** It has noSQL.

Reviewer: ```suggestion
**A:** It has NoSQL.
```
